docs/SOPs/archiving_SOP.md

```diff
@@ -368,6 +370,27 @@ bsub -J laurenti_upload -M 64000 'singularity run -B /nfs/production/hca/laurent
 
 If running parallel jobs, choose different <file_name> / <job_names> because you will have multiple file-upload-infos and multiple jobs, in this case I’ve used the name of the output bam file as the job_name and file_name.
 
+
+### Uploading files directly to ENA
```

Comment:
Hmm, I don't think this is ready to be put in this SOP for archiving. This step is not part of the current DSP integration. We should have the direct archiving full ENA archiving first before putting in the operational Archiving SOP (which still uses DSP).

Reply:
If it is not ready, than we don't merge it yet.

Reply:
or specify that it is not relevant until the task is finished.